### ISSUE TITLE
Does not work with Spatie NovaTranslatable

### ISSUE THREAD

**Issue by salierg** [eminiarts/nova-tabs#112]
This package show nothing when you try to display a translatable field in a tab.
We are using this package to manage translations for some fields:
https://github.com/spatie/nova-translatable

```
new Tabs('Tabs', [
                'Infos' => [
                    Translatable::make([
                        Text::make('Titre', 'title')->sortable()
                    ]),
                ],
            ]),
```

**Comment by RVxLab** [comment]
I'm gonna need a bit more info on this one. What exactly do you mean with "shows nothing"?

Could you provide a screenshot?

**Comment by iSaadSalman** [comment]
@RVxLab It shows loading on the screen but in debugger it shows following errors:

<img width="676" alt="Screen Shot 2020-11-03 at 4 16 19 PM" src="https://user-images.githubusercontent.com/736170/97989808-1d8bf480-1df0-11eb-9937-0126de3c0b96.png">

**Comment by RVxLab** [comment]
Thanks for the screenshot, I'll look further into it.

**Comment by RVxLab** [comment]
Sorry it's been a long time since I said I'd look into it.

With my latest changes (to be released in 1.4) this should be fixed. Could you verify this is now working on the master branch?

Thanks.

**Comment by bdsoha** [comment]
Doesn't work any translation.
